index.html

```diff
@@ -21,39 +27,29 @@
     <script src="https://js.arcgis.com/3.16/"></script>
     <script src="node_modules/esri-system-js/dist/esriSystem.js"></script>
   </head>
-
+  
   <!-- 3. Display the application -->
-  <body class="calcite">
-    <div class="container-fluid">
-      <my-app>Loading...</my-app>
-    </div>
+  <body>
+    <my-app>Loading...</my-app>
     <script>
-    // configure system.js
-    System.config({
-      packages: {
-        app: {
-          defaultExtension: 'js'
-        }
-      }
-    });
-    // load esri modules needed by this application
-    // into a System.js module called esri
-    esriSystem.register([
-      'esri/arcgis/utils',
-      'esri/basemaps',
-      'esri/dijit/Legend',
-      'esri/dijit/Search'
-    ], function () {
-      // then bootstrap application
-      System.import('app/boot')
-        .then(null, console.error.bind(console));
-    }, {
-      moduleNameOverrides: {
-        'esri/arcgis/utils': 'arcgisUtils',
-        'esri/basemaps': 'esriBasemaps'
-      }
-    });
-  </script>
+      // load esri modules needed by this application
```

Comment:
This should probably be moved into system.config.js now. That would also require it's script tag to be moved below the script tag for esriSystem.js